pegasus/sites.v3/code.org/public/privacy.md

```diff
@@ -18,80 +18,83 @@ Please read this Privacy Policy carefully before accessing or using the Website.
+Attendance at professional learning workshops | Participation in professional learning programs is optional.<br>For Teachers who participate, this attendance data will be stored. | Attendance of Teachers who attend our professional learning workshops is stored and associated with the teacher’s account on Code.org. <br><br>This data may be shared (along with the Teacher’s identity) with any other parties involved in the Teacher’s professional learning, such as the in-person facilitator who led the workshop, or the professional learning organization hosting the workshop, or the school district of the Teacher. In some cases, the school district may use the workshop attendance data to compensate Teachers for participating in the Code.org professional learning program.
+Progress, answers, documents, projects and peer reviews for online professional learning. | Participation in professional learning programs is optional.  <br>For Teachers who participate, this data will be stored. | Progress and answers in online professional-learning courses for Teachers is stored in their Teacher account in order to allow Teachers to pick up where they left off.<br><br>This includes the lesson plans, documents and other projects Teachers create as part of finishing the online learning courses. After submitting a document or project, Teachers receive peer feedback from each other which is also stored so that they can read it.<br>Teachers also take a self assessment survey to create a custom learning plan. The results of this survey are stored with the Teacher’s account along with their custom plan.
+Forum posts and profiles | Optional | For Teachers who choose to participate in the teacher support forums at forum.code.org, posts and replies are shared publicly. Teachers may also optionally supply additional information for their forum profile, including their profile picture, location, website, and “about me” text. This information is used in no other way beyond the forum itself.
 <br>
-\* A teacher account on Code.org has all the functionality of a student account, and as a result the data collected and stored for a teacher account is a superset of the data stored for a student account.
+\* A Teacher account on Code.org has all the functionality of a Student account, and as a result the data collected and stored for a Teacher account is a superset of the data stored for a Student account.
 
 \*\* Code.org partners with local non-profit professional learning organizations, our “Regional Partners.” We share limited information (as outlined in this privacy policy, and never including Student personal information) with these partners to allow them to run local professional development workshops and support teachers and schools in their region. Each Regional Partner signs an Agreement with Code.org that requires them to comply with this privacy policy and to treat any information received through the partnership as confidential.
```

Comment:
workshops and support `Teachers` and schools in their region?

Also change to professional learning?

Reply:
including Student `Personal Information`